.pre-commit-config.yaml

```diff
@@ -14,7 +14,7 @@ repos:
     rev: 3.7.9
     hooks:
       - id: flake8
-        additional_dependencies: [-e, git+git://github.com/pycqa/pyflakes.git@1911c20#egg=pyflakes]
+        additional_dependencies: [-e, 'git+git://github.com/pycqa/pyflakes.git@1911c20#egg=pyflakes']
```

Comment:
yaml can handle all sorts of crazy strings. did it actually cause a problem?

Reply:
Whenever a I committed without quotes it would give me a syntax error pointing to the colons. I added them and so it would work.

Reply:
what would give you the syntax error?

Reply:
Executing without quotes this gives an error.
`git commit -m "try without quotes"`

An error has occurred: InvalidConfigError: 
==> File .pre-commit-config.yaml
=====> while scanning a plain scalar
  in "<unicode string>", line 17, column 39
found unexpected ':'
  in "<unicode string>", line 17, column 46

Reply:
pre-commit seems to be installing the new environment every time I run it which seems like another bug.
I haven't used it in a while because it became too much pain.

You can upgrade the dev dependncies with:
```
pip install -r requirements/dev.txt  --upgrade
```

Reply:
It keeps going anyways. Maybe because of the OS(macos)?

Reply:
can you jump into the Hydra/OmegaConf chat?
https://hydra-framework.zulipchat.com/